acme/order.go

```diff
@@ -32,7 +32,7 @@ type Order struct {
 	// status (required, string):  The status of this order.  Possible
 	// values are "pending", "ready", "processing", "valid", and
 	// "invalid".  See Section 7.1.6.
-	Status string `json:"status"`
+	Status string `default:"pending";json:"status"`
```

Comment:
This is not valid and doesn't work as you think. The necessary changes are a little more involved than this.

Reply:
Can you help out?